google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -2532,6 +2634,88 @@ Blob create(
    */
   URL signUrl(BlobInfo blobInfo, long duration, TimeUnit unit, SignUrlOption... options);
 
+  /**
+   * Generates a URL and a map of fields that can be specified in a form to submit a POST request.
```

Comment:
`in an HTML form`
You can link to relevant documentation here: https://cloud.google.com/storage/docs/xml-api/post-object#usage_and_examples